.github/workflows/source-package.yml

```diff
@@ -1,30 +1,25 @@
 name: Build a source package
 
-# TODO: enable this workflow for pull_request (will require a different way to do the archive)
-on: [push]
+on: [push, pull_request]
 
 jobs:
   build:
 
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
 
     steps:
-        # Don't use checkout, we need a full clone for this workflow
-      - name: Clone repository
-        run: |
-          git clone https://github.com/yubico/yubikey-manager-qt
+      - uses: actions/checkout@v1
 
       - name: Build a .tar.gz source package
         run: |
           export REF=$(echo ${GITHUB_REF} | cut -d '/' -f 3)
-          cd yubikey-manager-qt
-          git checkout $REF
-          git archive --format=tar.gz -o yubikey-manager-qt-$REF.tar.gz --prefix yubikey-manager-qt/ $REF
+          cd ..
+          tar zcvf yubikey-manager-qt-$REF.tar.gz --exclude-vcs yubikey-manager-qt
```

Comment:
Why not use `git archive`?

Reply:
I did not find a good way to make it work on the `pull_request` event, but if there is a good way I'm open to it.